requirements.in

```diff
@@ -11,6 +11,11 @@ Flask==2.2.5
 flask-talisman==1.0.0
 flask-WTF==1.1.1
 
+#-----------------------------------
+#   AWS
+#-----------------------------------
+boto3
```

Comment:
Was this just erroneously absent before then?

Reply:
Sort of.

We depend on it directly (ie our app code imports boto3) so it should be a top-level dependency, not a transitive one.

It was previously being pulled in automatically by fsd-utils, so we got it "for free", but a recent change on fsd-utils to remove `flipper-client` (by default) means that we no longer get it for free. So we need to list it explicitly.